test/parallel/test-buffer-arraybuffer.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const Buffer = require('buffer').Buffer;
+const LENGTH = 16;
+
+const ab = new ArrayBuffer(LENGTH);
+const dv = new DataView(ab);
+const ui = new Uint8Array(ab);
+const buf = new Buffer(ab);
+
+
+assert.ok(buf instanceof Buffer);
+// For backwards compatibility of old .parent property, if the Buffer uses the
```

Comment:
Nit: I think this comment is confusing. Instead it's more like "Since `buf` is not a slice, it should have `.parent` of `undefined`." Right?